### PR TITLE
video drivers calfiy. input section

### PR DESCRIPTION
**Pull request by zlice**

### CODE REVIEW

File: src/config/xorg/index.md [hunk 1]
```diff
@@ -51,3 +51,35 @@ Void also provides proprietary video drivers, which are available in the
...
+| `xf86-input-libinput` |
+| `xf86-input-mtrack`   |
+| `xf86-input-synaptics`|
+| `xf86-input-vmmouse`  |
+| `xf86-input-wacom`    |
+
+## Xorg config
+
+An Xorg config for a keyboard may look something like a file
+`/etc/X11/xorg.conf.d/30-keyboard.conf` with the contents.
```
Comment (Vaelatern) [inline src/config/xorg/index.md]: Please emphasize that this is almost never necessary.

**Comment by bobertlo** [comment]
Thank you for splitting these up, but do try to make changes inside of existing PRs. Instead of doing a `git commit` you can just use `git commit --amend` to change your last commit and then `git push -f origin <branchname>` to force update. If you forget to do this and create a new commit you can run `git rebase -i HEAD~2` to merge the last two commits together.

File: src/config/xorg/index.md [hunk 2]
```diff
@@ -51,3 +51,35 @@ Void also provides proprietary video drivers, which are available in the
...
+
+```
+Section "InputClass"
+  Identifier "keyboard-all"
+  Driver "evdev"
+  MatchIsKeyboard "on"
+EndSection
+```
+
+> Note: For more in depth configurations, please seek other documentation.
```
Comment (bobertlo) [inline src/config/xorg/index.md]: I think this is unnecessary, unless a link to documentation is given. I would lean towards just removing this though.

**Comment by Vaelatern** [comment]
You can then run `git push -f` to force the PR to update

File: src/config/xorg/index.md (hunk 1 above)
Comment (zlice) [reply src/config/xorg/index.md]: added that the config is to use a specific driver

File: src/config/xorg/index.md (hunk 2 above)
Comment (zlice) [reply src/config/xorg/index.md]: removed

**Comment by zlice** [comment]
i'd love to know why (even with "conflicts") the rebase worked today and wouldn't the other day .-.

File: src/config/xorg/index.md (hunk 1 above)
Comment (bobertlo) [reply src/config/xorg/index.md]: Everything else looks great, but I would appreciate a stronger notice that this is almost never needed.

"Although it is almost never necessary, an Xorg config for ..."

**Pull request merged**